charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -73,6 +73,10 @@ spec:
             - --logtostderr
             - --v=5
           env:
+            - name: CSI_DRIVER_VERSION
+              value: {{ .Chart.AppVersion }}
```

Comment:
it would be better to refer to the driver.driverVersion IMO https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/Makefile#L21